Ui/Component/Listing/Column/EQ8Score.php

```diff
@@ -5,6 +5,7 @@
 use Magento\Framework\View\Element\UiComponent\ContextInterface;
 use Magento\Framework\View\Element\UiComponentFactory;
 use Magento\Ui\Component\Listing\Columns\Column;
+use Magento\Sales\Api\OrderRepositoryInterface;
```

Comment:
At one point these were all in alphabetical order, although that clearly changed before your update. Still, if you wouldn't mind, I think there is value in re-alphabetizing them.

Reply:
agreed